cmd/hosts-file-daemon/daemon.go

```diff
@@ -66,100 +67,177 @@ func Run(daemonConfig *DaemonConfig) {
 	go ManageIngressChanges(daemonConfig, updatesChannel, hostsFile)
 	go ManageServiceChanges(daemonConfig, updatesChannel, hostsFile)
 
+	go func() {
+		time.Sleep(time.Second * 60)
```

Comment:
Should probably be defined as a const, or at the very least just be defined outside this scope.